src/base/bootstrap-overrides/_core.scss

```diff
@@ -66,6 +67,22 @@ a {
 			padding-bottom: 0;
 		}
 	}
+	&.dt-max {
```

Comment:
This class is not used in the examples. They use `.max-dt`. Please change whichever accordingly.

Reply:
I have updated the utilities page to dt-max to align with dl-horizontal (front loading the element).  Thank you for catching that.